### ISSUE TITLE
es-la.json not have = symbol

### ISSUE THREAD

**Issue by vay3t** [hak5/bashbunny-payloads#481]
doing a proof of concept I realized that the "=" character is missing in es-la.json.

**Comment by rsxchin** [comment]
This was fixed in d56f4fd788a53acf7b1837da06443fbbdc5fe37b 
Please close this issue

**Comment by bad-antics** [comment]
Good catch. Looking at the `es-la.json` keyboard layout file, line 147 has `)` mapped where `=` should be.

**For the Latin American Spanish keyboard layout**, the `=` key is typically at the position that produces `)` on a US layout, so the mapping needs to be corrected.

**Workaround until a fix is merged:**

You can manually fix this on your Bash Bunny:
1. Put the Bunny in arming mode
2. Edit `languages/es-la.json` on the USB storage
3. Find line ~147 and change the `)` mapping to `=`

**Or specify the character directly in your payload:**
```bash
# Instead of relying on the language map for =
Q STRING "variable=value"
# Use the hex code if = isn't typing correctly
Q ALT STRING 061
```

This is a legitimate bug in the language mapping file. A PR fixing this specific character mapping would be a good contribution if anyone wants to submit one.

For reference, the correct HID scancode for `=` on an ES-LA keyboard depends on whether it requires a modifier key. On most Latin American layouts, `=` is `Shift+0` or at a dedicated key position that differs from US layout.
